### ISSUE TITLE
Multiple components not rendered with async data as props (reopened)

### ISSUE THREAD

**Issue by dinokajtez** [nuxt/nuxt.js#1208]
Hi,
i have problem with index page not rendered correctly when i access it directly (not from some other page). I am fetching data asynchronously and further i am using that data in three different components on the page. If I comment two of them page will be rendered correctly but as soon as I add another component (that was previously commented) the page breaks. Please refer to [this issue](https://github.com/nuxt/nuxt.js/issues/1185). [This is reproduction repo](https://github.com/dinokajtez/nuxt-js-reproduction) created as it was required by reviewer.

**Comment by alexchopin** [comment]
Hi @dinokajtez,
I use your reproduction project and a lot of Vue warnings appear.
It's due to SSR Hydration with Vue, that's why we have created Nuxt.js plugins.
For example in this [file](https://github.com/dinokajtez/nuxt-js-reproduction/blob/master/components/slider-big.vue#L29), try to create a [Nuxt.js plugin](https://nuxtjs.org/guide/plugins#client-side-only) to import vue-slick on client side only.

**Comment by dinokajtez** [comment]
Hi @alexchopin ,
thanks for quick response. I tried to implement what you suggested but with no luck. To be frank, i don't know how to make vue-slick work as plugin. But, all this made me wander what really happened as issue was still there also after I removed all slider components from the page.

So, I found the real cause of the issue. In third component  that is in the page data retrieved from backend contains '<p>' tags and that code i was binding to other '<p>' tag in template (and we all now that this is not valid HTML structure). After changing the parent from p to div it all works ok. But still, there are two Vue warnings from slider components and this does not break template.

Thanks again for help and if you know how to make vue-slick as plugin I am all ears.
